lib/transports/index.js

```diff
@@ -3,13 +3,20 @@ var templates = require('../templates')
 var t = require('../translations').t
 var log = require('debug')('democracyos:notifier:transports')
 
-function createTransport(config) {
+function createTransport (config) {
   log('Initializing nodemailer API client')
   var opts = config && config.service ? config : undefined
+  if (config && config.nodemailer) {
+    Object.keys(config.nodemailer)
+      .forEach(function (key) {
+        opts[key] = config.nodemailer[key]
+      })
+    delete config.nodemailer
```

Comment:
`config` object shouldn't be mutated. Maybe creating a new `opts` object before.